docs/production-deployment/cloud/aws-export-s3.mdx

```diff
@@ -0,0 +1,134 @@
+---
+id: aws-export-s3
+title: Exporting Workflow Event History to AWS S3
+sidebar_label: AWS Export to S3
+description: Export Workflow History to AWS S3
+slug: /cloud/export/aws-export-s3
+toc_max_heading_level: 4
+keywords:
+  - explanation
+  - how-to
+  - operations
+  - temporal cloud
+  - term
+  - verify
+  - workflow history export
+  - aws
+tags:
+  - explanation
+  - how-to
+  - operations
+  - temporal-cloud
+  - term
+  - verify
+  - workflow-history-export
+  - aws
+---
+
+### Prerequisites {#prerequisites}
```

Comment:
I might check with other pages but I would make this level H2 header since it looks a bit odd if you click the gcp one and its not consistent. I think the gcp one is larger.

Reply:
Sounds good. Copy pasta error